system_metrics_collector/src/topic_statistics_collector/received_message_period.hpp

```diff
@@ -23,9 +23,13 @@
 #include "system_metrics_collector/collector.hpp"
 
 #include "rcl/time.h"
-#include "rclcpp/clock.hpp"
 
 
+namespace
+{
+constexpr const int64_t kUninitializedTime{0};
```

Comment:
```suggestion
static constexpr const int64_t kUninitializedTime{0};
```

Reply:
I think `static` is redundant here since the anonymous namespace is going to make it file-scope.

Reply:
That was my thought as well concerning static.